frontend/src/v5/store/tickets/tickets.helpers.ts

```diff
@@ -15,8 +15,135 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { formatMessage } from '@/v5/services/intl';
 import { FederationsHooksSelectors } from '@/v5/services/selectorsHooks/federationsSelectors.hooks';
+import { trimmedString } from '@/v5/validation/shared/validators';
+import { isEmpty, isUndefined } from 'lodash';
+import * as Yup from 'yup';
+import { PropertyDefinition } from './tickets.types';
 
 export const modelIsFederation = (modelId: string) => (
 	!!FederationsHooksSelectors.selectContainersByFederationId(modelId).length
 );
+
+const MAX_TEXT_LENGTH = 50;
```

Comment:
Not sure where you got these numbers from?

In the backend,[ TEXT and LONG_TEXT](https://github.com/3drepo/3drepo.io/blob/staging/backend/src/v5/schemas/tickets/validators.js#L33) are of type `title` and `longDescription` [which are 120 and 1200 respectively.](https://github.com/3drepo/3drepo.io/blob/staging/backend/src/v5/utils/helper/yup.js#L70)

Reply:
I don't remember exactly why I put that to TEXT be have a maxLenght of 50, but I got the title length from the figma design 
![image](https://user-images.githubusercontent.com/43547703/198622087-dc564e3e-9247-49f7-a66c-cb2184308149.png)

Reply:
I got the title length from the figma design 
![image](https://user-images.githubusercontent.com/43547703/198622087-dc564e3e-9247-49f7-a66c-cb2184308149.png), but since it can span over multiple lines, I assumed that the title was a longText type, hence, the 120

As for the 50 chars length, I used this [validator](https://github.com/3drepo/3drepo.io/blob/ISSUE_3657/frontend/src/v5/store/tickets/tickets.helpers.ts#L27) without realising that was not the actual length and I just kept it

Reply:
title is of type strings.title https://github.com/3drepo/3drepo.io/blob/staging/backend/src/v5/schemas/tickets/index.js#L138

So if anything it falls in the `TEXT` type, nothing to do with  `LONG_TEXT`

Reply:
backend validators are always made under consultation with the product team which is more probably reliable the source of truth than anything else

if there's any questions/doubts, just look it up on the code or ask me or christos!